configuration/white_labels/__init__.py

```diff
@@ -0,0 +1,11 @@
+from ._default import DefaultConfiguarationData
+from .base import ConfigurationData
+from .co import CoConfiguarationData
+from .nc import NcConfiguarationData
+
+
+white_label_config: dict[str, ConfigurationData] = {
+    "co": CoConfiguarationData,
+    "nc": NcConfiguarationData,
+    "_default": DefaultConfiguarationData,
```

Comment:
What do you think about fixing the `Configuration` spelling on all of these except lines 2 & 7?